landing-pages/site/content/en/survey/_index.html

```diff
@@ -1,23 +1,18 @@
 ---
-title: "Airflow Survey 2023"
-linkTitle: "Airflow Survey 2023"
+title: "Airflow Survey 2024"
+linkTitle: "Airflow Survey 2024"
```

Comment:
I'm not sure we should outright replace it. Going back and looking at old surveys is helpful.

To be honest, this should have been brought up when 2023 was added this way, but the second best time to bring it up is now I guess.

Reply:
Maybe we move the old content to a separate page and add links at the bottom, something like:

```
Past surveys:
- 2023
- 2022
- 2020
- 2019
```

Reply:
> Maybe we move the old content to a separate page and add links at the bottom, something like:
> 
> ```
> Past surveys:
> - 2023
> - 2022
> - 2020
> - 2019
> ```

I like this idea. This will be cleaner structure.

Reply:
Big plus one for this! I was looking back for historical survey data recently and it was a pain that it wasn't all in one place (blogs, website, dev list). I love the idea of a page with all the historical survey results linked in one place :heart:

Reply:
I have added previous surveys.